### ISSUE TITLE
Static Media Failure / 404 Errors

### ISSUE THREAD

**Issue by spirrello** [bootc/netbox-chart#18]
Hi,

I'm getting 404s when I try to browse to the Netbox app.  Can you point me on where to look first?

Page directs to this:
https://MYSITE/media-failure/?filename=css/base.css

If I curl the endpoint from within my K8s cluster I get this:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <title>Home - NetBox</title>
    <link rel="stylesheet"
          href="/static/bootstrap-3.4.1-dist/css/bootstrap.min.css"
          onerror="window.location='/media-failure/?filename=bootstrap-3.4.1-dist/css/bootstrap.min.css'">
    <link rel="stylesheet"
          href="/static/font-awesome-4.7.0/css/font-awesome.min.css"
          onerror="window.location='/media-failure/?filename=font-awesome-4.7.0/css/font-awesome.min.css'">
    <link rel="stylesheet"
          href="/static/jquery-ui-1.12.1/jquery-ui.css"
          onerror="window.location='/media-failure/?filename=jquery-ui-1.12.1/jquery-ui.css'">
    <link rel="stylesheet"
          href="/static/select2-4.0.12/dist/css/select2.min.css"
          onerror="window.location='/media-failure/?filename=select2-4.0.12/dist/css/select2.min.css'">
    <link rel="stylesheet"
          href="/static/select2-bootstrap-0.1.0-beta.10/select2-bootstrap.min.css"
          onerror="window.location='/media-failure/?filename=select2-bootstrap-0.1.0-beta.10/select2-bootstrap.min.css'">
    <link rel="stylesheet"
          href="/static/flatpickr-4.6.3/themes/light.css"
          onerror="window.location='/media-failure/?filename=flatpickr-4.6.3/themes/light.css'">
    <link rel="stylesheet"
          href="/static/css/base.css?v2.8.1"
          onerror="window.location='/media-failure/?filename=css/base.css'">
    <link rel="icon" type="image/png" href="/static/img/netbox.ico" />
    <meta charset="UTF-8">
    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
</head>
<body>


<nav class="navbar navbar-default navbar-fixed-top">
    <div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">
                <img src="/static/img/netbox_logo.svg" height="30" />
            </a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">

            <ul class="nav navbar-nav">
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Organization <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Sites</li>
                        <li class="disabled">

                            <a href="/dcim/sites/">Sites</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/regions/">Regions</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Racks</li>
                        <li class="disabled">

                            <a href="/dcim/racks/">Racks</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/rack-groups/">Rack Groups</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/rack-roles/">Rack Roles</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/rack-reservations/">Reservations</a>
                        </li>
                        <li class="disabled">
                            <a href="/dcim/rack-elevations/">Elevations</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Tenancy</li>
                        <li class="disabled">

                            <a href="/tenancy/tenants/">Tenants</a>
                        </li>
                        <li class="disabled">

                            <a href="/tenancy/tenant-groups/">Tenant Groups</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Tags</li>
                        <li class="disabled">
                            <a href="/extras/tags/">Tags</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Devices <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Devices</li>
                        <li class="disabled">

                            <a href="/dcim/devices/">Devices</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/device-roles/">Device Roles</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/platforms/">Platforms</a>
                        </li>
                        <li class="disabled">
                            <a href="/dcim/virtual-chassis/">Virtual Chassis</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Device Types</li>
                        <li class="disabled">

                            <a href="/dcim/device-types/">Device Types</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/manufacturers/">Manufacturers</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Inventory</li>
                        <li class="disabled">

                            <a href="/dcim/inventory-items/">Inventory Items</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Connections</li>
                        <li class="disabled">

                            <a href="/dcim/cables/">Cables</a>
                        </li>
                        <li class="disabled">
                            <a href="/dcim/console-connections/">Console Connections</a>
                        </li>
                        <li class="disabled">
                            <a href="/dcim/power-connections/">Power Connections</a>
                        </li>
                        <li class="disabled">
                            <a href="/dcim/interface-connections/">Interface Connections</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Device Components</li>
                        <li class="disabled">

                            <a href="/dcim/interfaces/">Interfaces</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/front-ports/">Front Ports</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/rear-ports/">Rear Ports</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/console-ports/">Console Ports</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/console-server-ports/">Console Server Ports</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/power-ports/">Power Ports</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/power-outlets/">Power Outlets</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/device-bays/">Device Bays</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">IPAM <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">IP Addresses</li>
                        <li class="disabled">

                            <a href="/ipam/ip-addresses/">IP Addresses</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Prefixes</li>
                        <li class="disabled">

                            <a href="/ipam/prefixes/">Prefixes</a>
                        </li>
                        <li class="disabled">

                            <a href="/ipam/roles/">Prefix/VLAN Roles</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Aggregates</li>
                        <li class="disabled">

                            <a href="/ipam/aggregates/">Aggregates</a>
                        </li>
                        <li class="disabled">

                            <a href="/ipam/rirs/">RIRs</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">VRFs</li>
                        <li class="disabled">

                            <a href="/ipam/vrfs/">VRFs</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">VLANs</li>
                        <li class="disabled">

                            <a href="/ipam/vlans/">VLANs</a>
                        </li>
                        <li class="disabled">

                            <a href="/ipam/vlan-groups/">VLAN Groups</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Services</li>
                        <li class="disabled">

                            <a href="/ipam/services/">Services</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Virtualization <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Virtual Machines</li>
                        <li class="disabled">

                            <a href="/virtualization/virtual-machines/">Virtual Machines</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Clusters</li>
                        <li class="disabled">

                            <a href="/virtualization/clusters/">Clusters</a>
                        </li>
                        <li class="disabled">

                            <a href="/virtualization/cluster-types/">Cluster Types</a>
                        </li>
                        <li class="disabled">

                            <a href="/virtualization/cluster-groups/">Cluster Groups</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Circuits <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Circuits</li>
                        <li class="disabled">

                            <a href="/circuits/circuits/">Circuits</a>
                        </li>
                        <li class="disabled">

                            <a href="/circuits/circuit-types/">Circuit Types</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Providers</li>
                        <li class="disabled">

                            <a href="/circuits/providers/">Providers</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Power <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Power</li>
                        <li class="disabled">

                            <a href="/dcim/power-feeds/">Power Feeds</a>
                        </li>
                        <li class="disabled">

                            <a href="/dcim/power-panels/">Power Panels</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Secrets <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Secrets</li>
                        <li class="disabled">

                            <a href="/secrets/secrets/">Secrets</a>
                        </li>
                        <li class="disabled">

                            <a href="/secrets/secret-roles/">Secret Roles</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown">
                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Other <span class="caret"></span></a>
                    <ul class="dropdown-menu">
                        <li class="dropdown-header">Logging</li>
                        <li class="disabled">
                            <a href="/extras/changelog/">Change Log</a>
                        </li>
                        <li class="divider"></li>
                        <li class="dropdown-header">Miscellaneous</li>
                        <li class="disabled">

                            <a href="/extras/config-contexts/">Config Contexts</a>
                        </li>
                        <li class="disabled">
                            <a href="/extras/scripts/">Scripts</a>
                        </li>
                        <li class="disabled">
                            <a href="/extras/reports/">Reports</a>
                        </li>
                    </ul>
                </li>

            </ul>

            <ul class="nav navbar-nav navbar-right">



                        <li><a href="/login/?next=/"><i class="fa fa-sign-in"></i> Log in</a></li>


            </ul>
            <form action="/search/" method="get" class="navbar-form navbar-right" id="navbar_search" role="search">
                <div class="input-group">
                    <input type="text" name="q" class="form-control" placeholder="Search">
                    <span class="input-group-btn">
                        <button type="submit" class="btn btn-primary">
                            <i class="fa fa-search"></i>
                        </button>
                    </span>
                </div>
            </form>
        </div>
    </div>
</nav>

    <div class="container wrapper">








<div class="row" style="padding-bottom: 20px">
    <div class="col-md-12 text-center">
        <form action="/search/" method="get" class="form-inline">
            <input type="text" name="q" value="" placeholder="Search" id="id_q" class="form-control" style="width: 350px" />
            <select name="obj_type" class="form-control" placeholder="Type" id="id_obj_type">
  <option value="" selected>All Objects</option>

  <optgroup label="Circuits">
  <option value="provider">Providers</option>

  <option value="circuit">Circuits</option>

  </optgroup>
  <optgroup label="DCIM">
  <option value="site">Sites</option>

  <option value="rack">Racks</option>

  <option value="rackgroup">Rack Groups</option>

  <option value="devicetype">Device types</option>

  <option value="device">Devices</option>

  <option value="virtualchassis">Virtual Chassis</option>

  <option value="cable">Cables</option>

  <option value="powerfeed">Power Feeds</option>

  </optgroup>
  <optgroup label="IPAM">
  <option value="vrf">VRFs</option>

  <option value="aggregate">Aggregates</option>

  <option value="prefix">Prefixes</option>

  <option value="ipaddress">IP addresses</option>

  <option value="vlan">VLANs</option>

  </optgroup>
  <optgroup label="Secrets">
  <option value="secret">Secrets</option>

  </optgroup>
  <optgroup label="Tenancy">
  <option value="tenant">Tenants</option>

  </optgroup>
  <optgroup label="Virtualization">
  <option value="cluster">Clusters</option>

  <option value="virtualmachine">Virtual machines</option>

  </optgroup>
</select>
            <button type="submit" class="btn btn-primary">Search</button>
        </form>
    </div>
</div>

<div class="row">
    <div class="col-sm-6 col-md-4">
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Organization</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Sites</h4>

                    <p class="list-group-item-text text-muted">Geographic locations</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Tenants</h4>

                        <p class="list-group-item-text text-muted">Customers or departments</p>
                    </div>
            </div>
        </div>
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>DCIM</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Racks</h4>

                    <p class="list-group-item-text text-muted">Equipment racks, optionally organized by group</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Device Types</h4>

                    <p class="list-group-item-text text-muted">Physical hardware models by manufacturer</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Devices</h4>

                    <p class="list-group-item-text text-muted">Rack-mounted network equipment, servers, and other devices</p>
                </div>
                <div class="list-group-item">
                    <h4 class="list-group-item-heading">Connections</h4>

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <p style="padding-left: 20px;">Cables</p>


                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <p style="padding-left: 20px;">Interfaces</p>


                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <p style="padding-left: 20px;">Console</p>


                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <p style="padding-left: 20px;">Power</p>

                </div>
            </div>
        </div>
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Power</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Power Feeds</h4>

                    <p class="list-group-item-text text-muted">Electrical circuits delivering power from panels</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Power Panels</h4>

                    <p class="list-group-item-text text-muted">Electrical panels receiving utility power</p>
                </div>
            </div>
        </div>
    </div>
    <div class="col-sm-6 col-md-4">
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>IPAM</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">VRFs</h4>

                    <p class="list-group-item-text text-muted">Virtual routing and forwarding tables</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Aggregates</h4>

                    <p class="list-group-item-text text-muted">Top-level IP allocations</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Prefixes</h4>

                    <p class="list-group-item-text text-muted">IPv4 and IPv6 network assignments</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">IP Addresses</h4>

                    <p class="list-group-item-text text-muted">Individual IPv4 and IPv6 addresses</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">VLANs</h4>

                    <p class="list-group-item-text text-muted">Layer two domains, identified by VLAN ID</p>
                </div>
            </div>
        </div>
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Circuits</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Providers</h4>

                    <p class="list-group-item-text text-muted">Organizations which provide circuit connectivity</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Circuits</h4>

                    <p class="list-group-item-text text-muted">Communication links for Internet transit, peering, and other services</p>
                </div>
            </div>
        </div>
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Virtualization</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Clusters</h4>

                    <p class="list-group-item-text text-muted">Clusters of physical hosts in which VMs reside</p>
                </div>
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Virtual Machines</h4>

                    <p class="list-group-item-text text-muted">Virtual compute instances running inside clusters</p>
                </div>
            </div>
        </div>
    </div>
    <div class="col-sm-6 col-md-4">
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Secrets</strong>
            </div>
            <div class="list-group">
                <div class="list-group-item">

                        <span class="badge pull-right"><i class="fa fa-lock"></i></span>
                        <h4 class="list-group-item-heading">Secrets</h4>

                    <p class="list-group-item-text text-muted">Cryptographically secured secret data</p>
                </div>
            </div>
        </div>
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Reports</strong>
            </div>

                <div class="panel-body text-muted">
                    <i class="fa fa-lock"></i> No permission
                </div>

        </div>
        <div class="panel panel-default">
            <div class="panel-heading">
                <strong>Change Log</strong>
            </div>

                <div class="panel-body text-muted">
                    <i class="fa fa-lock"></i> No permission
                </div>

        </div>
    </div>
</div>

        <div class="push"></div>

    </div>
    <footer class="footer">
        <div class="container">
            <div class="row">
                <div class="col-xs-4">
                    <p class="text-muted">netbox-7955d7fddb-494nb (v2.8.1)</p>
                </div>
                <div class="col-xs-4 text-center">
                    <p class="text-muted">2020-05-06 02:35:14 UTC</p>
                </div>
                <div class="col-xs-4 text-right noprint">
                    <p class="text-muted">
                        <i class="fa fa-fw fa-book text-primary"></i> <a href="http://netbox.readthedocs.io/">Docs</a> &middot;
                        <i class="fa fa-fw fa-cloud text-primary"></i> <a href="/api/docs/">API</a> &middot;
                        <i class="fa fa-fw fa-code text-primary"></i> <a href="https://github.com/netbox-community/netbox">Code</a> &middot;
                        <i class="fa fa-fw fa-support text-primary"></i> <a href="https://github.com/netbox-community/netbox/wiki">Help</a>
                    </p>
                </div>
            </div>
        </div>
    </footer>
<script src="/static/jquery/jquery-3.4.1.min.js"
        onerror="window.location='/media-failure/?filename=jquery/jquery-3.4.1.min.js'"></script>
<script src="/static/jquery-ui-1.12.1/jquery-ui.min.js"
        onerror="window.location='/media-failure/?filename=jquery-ui-1.12.1/jquery-ui.min.js'"></script>
<script src="/static/bootstrap-3.4.1-dist/js/bootstrap.min.js"
        onerror="window.location='/media-failure/?filename=bootstrap-3.4.1-dist/js/bootstrap.min.js'"></script>
<script src="/static/select2-4.0.12/dist/js/select2.min.js"
        onerror="window.location='/media-failure/?filename=select2-4.0.12/dist/js/select2.min.js'"></script>
<script src="/static/clipboard.js/clipboard-2.0.4.min.js"
        onerror="window.location='/media-failure/?filename=clipboard.js/clipboard-2.0.4.min.js'"></script>
<script src="/static/flatpickr-4.6.3/flatpickr.min.js"
        onerror="window.location='/media-failure/?filename=flatpickr-4.6.3/flatpickr.min.js'"></script>
<script src="/static/js/forms.js?v2.8.1"
        onerror="window.location='/media-failure/?filename=js/forms.js'"></script>
<script type="text/javascript">
    var netbox_api_path = "/api/";
    var loading = $(".loading");
    $(document).ajaxStart(function() {
        loading.show();
    }).ajaxStop(function() {
        loading.hide();
    });
</script>

</body>
</html>
```

**Comment by spirrello** [comment]
I made a little progress by changing the basepath to `/netbox`.  Now I'm able to render a page but still not completely functional.

![image](https://user-images.githubusercontent.com/7174260/81171500-7aeed800-8f6a-11ea-80e5-47aec2a98d41.png)

**Comment by spirrello** [comment]
Got it figured out after banging my head long enough.  Here's the solution and please let me know if there's a better way.  Side note, I'm using a basePath of `netbox/`:

Added these in the Values file:
```
basePath: 'netbox/'

healthCheckPath:
  netbox: '/netbox/login'
  nginx: '/netbox/'

```

In the configmap I changed this to include the basePath.

```
location /{{ .Values.basePath }}static/ {
          alias /opt/netbox/netbox/static/;
        }
```

Updated the healthchecks for the pods:
```
Netbox container
readinessProbe:
            httpGet:
              path: {{ .Values.healthCheckPath.netbox }}


Nginx container

readinessProbe:
            httpGet:
              path: {{ .Values.healthCheckPath.nginx }}
```

**Comment by bootc** [comment]
It sounds like my chart really could do with some improvements to cater for different `basePath` values, so I'm reopening this.

**Comment by bootc** [comment]
This is fixed in chart version 2.2.0.
